examples/tutorials/earth-relief.py

```diff
@@ -21,8 +21,9 @@
 
 ########################################################################################
 # Load sample Earth relief data for the entire globe at a resolution of 30 arc minutes.
-# The other available resolutions are show at :gmt-docs:`datasets/remote-data.html#global-earth-relief-grids`.
-grid = pygmt.datasets.load_earth_relief(resolution="30m")
+# The other available resolutions are show
+# at :gmt-docs:`datasets/remote-data.html#global-earth-relief-grids`.
+grid = pygmt.datasets.load_earth_relief(resolution="01d", registration="gridline")
```

Comment:
What's the reason to use the gridline-registered grid?
```suggestion
grid = pygmt.datasets.load_earth_relief(resolution="01d")
```

Reply:
You had mentioned that the gridline version was the cached file.